cpu/arm7_common/arm7_init.c

```diff
@@ -64,6 +64,7 @@ static inline void _init_data(void)
 
     dst = &__stack_usr_start;
     end = &__stack_end;
+    /* cppcheck-suppress comparePointers */
     while (dst < end) {
```

Comment:
I guess
```suggestion
    while ((uintptr_t)dst < (uintptr_t)end) {
```
would have worked too?`

Reply:
Maybe. But I think it is easier to read without the cast.